CoreWordCloudView/src/au/gov/asd/tac/constellation/views/wordcloud/phraseanalysis/PhraseAnalysisModelLoader.java

```diff
@@ -92,24 +91,14 @@ private static void processLines(final BufferedReader reader, final Map<String,
                     currentSet.addAll(map.get(line));
                     break;
                 case ADD_CHARACTERS:
-                    currentSet.add(singleCharacterLines ? line.charAt(0) : line);
-                    break;
-                case ADD_UNICODE_RANGE:
-                    final String rangeStart = line.substring(line.indexOf(OPEN_SET) + 1, line.indexOf(CLOSE_SET));
-                    final String rangeEnd = line.substring(line.indexOf(OPEN_SET, line.indexOf(CLOSE_SET) + 1) + 1, line.indexOf(CLOSE_SET, line.indexOf(CLOSE_SET)) + 1);
-                    final int[] range = {Integer.parseInt(rangeStart, 16), Integer.parseInt(rangeEnd, 16)};
-                    currentSet.add(range);
+                    currentSet.add(singleCharacterLines ? String.valueOf(line.charAt(0)) : line);
```

Comment:
I think in this case, `line.substring(0,1)` might be better (rather than converting to char and then back to string)